client/src/components/feedback/edit-question-for-queue-modal.tsx

```diff
@@ -255,7 +251,9 @@ function EditQuestionForQueueModal(props: {
                   component="span"
                   disabled={
                     selectedSubject == undefined ||
```

Comment:
this can all be shortened to 
`!selectedSubject || !selectedCategory || !customQuestion`